template.yml

```diff
@@ -42,6 +42,7 @@ Resources:
       AliasTarget:
         DNSName: !GetAtt DomainName.RegionalDomainName
         HostedZoneId: !GetAtt DomainName.RegionalHostedZoneId
+
```

Comment:
As a last step, I'm going to delete the template.yml file so there's one source of truth. Until then, leaving this here so folks can compare against the .erb file if needed.